contracts/identity/lib.rs

```diff
@@ -1,14 +1,51 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use ink::storage::traits::StorageLayout;
+
+/// Each identity will be associated with a unique identifier called `IdentityNo`.
+pub type IdentityNo = u64;
+
+/// We want to keep the address type very generic since we want to support any
+/// address format. We won't actually keep the addresses in the contract itself.
+/// Before storing them, we'll encrypt them to ensure privacy.
+pub type Address = Vec<u8>;
```

Comment:
I think we should limit the length of an Address

Reply:
#8